### PR TITLE
GenZ-108: Added a solution for the missing element task.

### PR DESCRIPTION
**Pull request by Alhasan-AJ**

### CODE REVIEW

File: examples/alhasan_work_area/fibonacci.py [hunk 1]
```diff
@@ -14,4 +14,16 @@ def fibonacci(series):
...
 
-print(fibonacci(10))   
\ No newline at end of file
+print(fibonacci(10))   
+
+
+def missing_element(lst):
+    expected_sum_of_list = 0
+    actual_sum_of_list = 0
+    for i in range(lst[0],lst[-1]+1):
```
Comment (CodelineAtyab) [inline examples/alhasan_work_area/fibonacci.py:23]: PEP-8: Lets put a space after ,

File: examples/alhasan_work_area/missing_element.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+def missing_element(lst):
+    expected_sum_of_list = 0
+    actual_sum_of_list = 0
+    for i in range(lst[0],lst[-1]+1):
```
Comment (CodelineAtyab) [inline examples/alhasan_work_area/missing_element.py:4]: PEP-8: Lets put a space after ,

File: examples/alhasan_work_area/missing_element.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+        expected_sum_of_list = expected_sum_of_list + i
+    for n in lst:
+        actual_sum_of_list += n
+    return (expected_sum_of_list - actual_sum_of_list)
+
+print(missing_element([1,2,3,5,6]))
```
Comment (CodelineAtyab) [inline examples/alhasan_work_area/missing_element.py:10]: PEP-8: Lets put a space after ,

**Review by ziyadabd — DISMISSED** [review]

**Review by CodelineAtyab — CHANGES_REQUESTED** [review]
Please see the comments

**Review by sarah-alshukaily — APPROVED** [review]

**Review by Sughiya-AlSaid — APPROVED** [review]

File: examples/alhasan_work_area/missing_element.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+def missing_element(lst):
```
Comment (CodelineAtyab) [inline examples/alhasan_work_area/missing_element.py:1]: Please consider adding a multiline comment to describe what this module is doing. For Example:
"""
This module contains a function that takes a list of unique integers in a specific range as input and returns the missing element as an output
"""

File: examples/alhasan_work_area/missing_element.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    expected_sum_of_list = 0
+    actual_sum_of_list = 0
+    for i in range(lst[0], lst[-1]+1):
+        expected_sum_of_list += i
+    for n in lst:
```
Comment (CodelineAtyab) [inline examples/alhasan_work_area/missing_element.py:6]: Please rename "n" OR put an inline comment that tells the devs, what n could be. Is it an integer coming from lst or something else ?

**Review by CodelineAtyab — CHANGES_REQUESTED** [review]
Great work using meaningful variable names. However there are a few suggestions. Please respond to those

**Review by Abbas-Abdulrab — APPROVED** [review]

**Review by CodelineAtyab — APPROVED** [review]
Great Logic!

**Review by aljab017 — APPROVED** [review]

**Review by sarah-alshukaily — APPROVED** [review]

**Pull request merged**